source/triggers/aws-eventbridge.txt

```diff
@@ -152,13 +171,363 @@ Procedure
+            :ref:`Error Handler Parameters <eventbridge-error-handler-parameters>`
+            section on this page.
+
+            Click :guilabel:`Run` to run the test. 
+
+         .. step:: Save the Function
+
+            Once you are satisfied with the custom error handler, click
+            :guilabel:`Save`. 
+
+   .. tab::
+      :tabid: cli
+
+      In order to update your trigger's configuration with an error handler, 
+      follow these steps to :ref:`Update an App  <deploy-cli>`. When you
+      update your configuration files in Step 3, do the following:
+
+      .. procedure::
+
+         .. step:: Write the Error Handler
+
+            Follow the steps in :ref:`Define a Function <define-a-function>` 
+            to write your error handler source code and configuration file. 
+
+            For the error handler source code, see the following template error handler:
+
+            .. code-block:: js
+               :caption: <functionName>.js
+               
+               exports = async function(error, changeEvent) {
+                  // This sample function will log additional details if the error is not
+                  // a DOCUMENT_TOO_LARGE error
+                  if (error.code === 'DOCUMENT_TOO_LARGE') {
+                     console.log('Document too large error');
+
+                     // Comment out the line below in order to skip this event and not suspend the Trigger
+                     throw new Error(`Encountered error: ${error.code}`);
+                  }
+
+                  console.log('Error sending event to EventBridge');
+                  console.log(`DB: ${changeEvent.ns.db}`);
+                  console.log(`Collection: ${changeEvent.ns.coll}`);
+                  console.log(`Operation type: ${changeEvent.operationType}`);
+
+                  // Throw an error in your function to suspend the trigger and stop processing additional events
+                  throw new Error(`Encountered error: ${error.message}`);
+               };
+
+         .. step:: Add an Error Handler to Your Trigger Configuration
+
+            Add an ``error_handler`` attribute to your trigger configuration file
+            in the ``Triggers`` folder. The trigger configuration file should 
+            resemble the following:
+
+            .. code-block:: json
+               :emphasize-lines: 13-18
+               :caption: <triggerName>.json
+
+                     {
+                     "name": "...",
+                     "type": "...",
+                     "event_processors": {
+                        "AWS_EVENTBRIDGE": {
+                              "config": {
+                                 "account_id": "<AWS Account ID>",
+                                 "region": "<AWS Region>",
+                                 "extended_json_enabled": <boolean>
+                              }
+                           }
+                        },
+                        "error_handler": {
```

Comment:
I think this indentation level might be off -- it is making the `error_handler` look like it is at the same level as `AWS_EVENTBRIDGE` (and within event_processors) but it should be at the same level as event_processors